doc/pi_setup.md

```diff
@@ -0,0 +1,107 @@
+# Raspberry Pi 3 setup
+
+## Create the Rasbpian Boot Disk
+Use [Apple Pi Baker](https://www.tweaking4all.com/software/macosx-software/macosx-apple-pi-baker/) to create a boot disk from the [Raspbian Jessie Lite](https://downloads.raspberrypi.org/raspbian_lite_latest) image.
+
+## Configuring SSH
+
+Insert the SD card into your development machine and add a file named `ssh` to SD card's root directory.
+
+Plug the Pi into the ethernet and find its ip address.
+
+You can then ssh into the machine:
+
+```
+$ ssh pi@<ip address>
+```
+
+The default password for this user is `raspbian`.
+
+
+## Update/Upgrade the System
+```
+$ sudo apt-get update
+$ sudo apt-get upgrade
+```
+
+## Setting up Wi-Fi
+```
+$ echo "network={\n\tssid=\"<ssid>\" psk=\"<wifi password>\"}" | sudo tee -a /etc/wpa_supplicant/wpa_supplicant.conf
+```
+
+
+
+### System Setup
+
+```shell
+sudo apt-get update
+sudo apt-get dist-upgrade
+sudo reboot
+
+sudo raspi-config
+sudo reboot
+
+sudo apt-get install vim screen nodejs libcurl4-openssl-dev git libreadline6-dev libssl-dev libyaml-dev libxml2-dev libxslt-dev autoconf ncurses-dev automake libtool bison
+echo "startup_message off" > ~/.screenrc
+echo "gem: --no-document" >> ~/.gemrc
+
+echo "export RAILS_ENV=production" >> ~/.bashrc
+echo "export CDPATH=.:/app/bender" >> ~/.bashrc
+echo "export PATH=./bin:./.bin:$PATH" >> ~/.bashrc
+echo "export EDITOR=/usr/bin/vim" >> ~/.bashrc
+
+
+sudo mkdir /app
+sudo chown pi:pi /app
+
+wget https://cache.ruby-lang.org/pub/ruby/2.2/ruby-2.2.5.tar.gz
+tar xzf ruby-2.2.5.tar.gz 
+cd ruby-2.2.5/
+./configure --prefix=/usr/local --enable-shared --disable-install-doc
+make -j 4
+sudo make install
+
+cd
+sudo gem i bundler
+sudo apt-get install postgresql libpq-dev
+sudo vi /etc/postgresql/9.4/main/pg_hba.conf
+# change "peer" to "trust"
+
+sudo /etc/init.d/postgresql reload
+psql -U postgres
+# CREATE ROLE bender_production;
+# CREATE DATABASE bender_production;
+
+git clone https://github.com/mruby/mruby.git
+cd mruby/
+git checkout 1.2.0
+./minirake
+./bin/mruby -v
+
+# Not sure where these are from
+bunzip2 -c bender_production_2016-07-07.sql.bz2 | psql -U postgres bender_production
+
+
+# Set up Rails app
+cd /app
+bundle install
+sudo gem i passenger
+
+sudo passenger-install-nginx-module
+wget http://downloads.sourceforge.net/project/pcre/pcre/8.32/pcre-8.32.tar.gz
+
+# Is this needed?
+sha256sum pcre-8.32.tar.gz 
+mv pcre-8.32.tar.gz pcre.tar.gz
+cd /usr/local/lib/ruby/gems/2.2.0/gems/passenger-5.1.7
```

Comment:
Here up was stuff to fix a botched passenger gem. `sudo passenger-install-nginx-module` is usually all that is needed